### ISSUE TITLE
As an EVCM previewing the tool, I want to see the map at a national level when I first visit the tool so that the first thing I see is clearly and recognizably is the US

### ISSUE THREAD

**Issue by katherinedm-usds** [usds/justice40-tool#29]
**Description**
The intent of this user story is to make explicit that when someone loads CEJST the first time, the first thing they see is a national level view as opposed to an empty space or arbitrary space, etc.

Update: We will create a separate ticket to explore the best way to represent Alaska and Hawaii and territories at the national zoom level so that those community members know they are represented even if they are not in the lower 48. Users will still be able to drag over and find these locations in the map populated with data.

Note, it's likely and needs to be confirmed that the federal programs impacted by Justice40 have funding relevant to the five major US territories but that's not needed for initial user research version of CEJST. 

**Solution**
 - [ ] First page load shows a national level map (geographic data only)
 - [ ] Would be good for initial version of CEJST for user research to show live data on national map for first page load, but if this is difficult to do in the near term for performance, etc. reasons it's open to discussion

**Describe alternatives you've considered**
N/A

**Links to user research or other resources**
User journey board https://app.mural.co/t/kameronkerger2998/m/kameronkerger2998/1619097401497/d8d8b090c94770d1d72c9c250db564b37676cc69

**Definition of "Done"**
When EVCM visits URL for the first time, they see a map of the US with CBG boundaries at an appropriate zoom level

**Comment by danhammer** [comment]
I think that we may have a solution for this issue through a clever tiling system.  See [`justice-map`](https://github.com/earthrise-media/justice-map).  The interactive chart may also address #31.  Note that this is a draft project.  We will add some design, docs, and interactive features soon: https://earthrise-media.github.io/justice-map

**Comment by switzersc-usds** [comment]
Next step here: Design+dev sync on an initial map view that includes Alaska, Hawaii, and territories. @KameronKerger will schedule this; it may be combinable with another design+dev sync happening.

**Comment by switzersc-usds** [comment]
Latest update here: We will use the Census TIGER data in vector format for the base map. Currently waiting on information from the geoplatform.gov folks on how long it will take them to add this to their pipeline -- should hear back by COB Friday 6/4.

**Comment by switzersc-usds** [comment]
@NatHillardUSDS @esfoobar-usds This is pretty related to https://app.zenhub.com/workspaces/justice40-60993f6e05473d0010ec44e3/issues/usds/justice40-tool/100, which is in review/QA. Does that mean this also is in Review/QA, or even ready to close?

**Comment by NatHillardUSDS** [comment]
correct, just moved!

**Comment by switzersc-usds** [comment]
Dev and design are syncing to review this Friday 6/18. @NatHillardUSDS owning scheduling sync. This will move to done hopefully after!

**Comment by switzersc-usds** [comment]
@NatHillardUSDS status here after Friday's sync with design? :)
